playwright.config.js

```diff
@@ -0,0 +1,81 @@
+// @ts-check
```

Comment:
Same for this file: there is already a `playwright.config.ts` so this shouldn't be needed.